wrappers/src/sync_session_cs.cpp

```diff
@@ -95,5 +113,44 @@ REALM_EXPORT void realm_syncsession_destroy(SharedSyncSession* session)
 {
     delete session;
 }
+    
+REALM_EXPORT void realm_install_syncsession_callbacks(decltype(s_refresh_access_token_callback) refresh_callback, decltype(s_session_error_callback) session_error_callback, decltype(s_progress_callback) progress_callback)
+{
+    s_refresh_access_token_callback = refresh_callback;
+    s_session_error_callback = session_error_callback;
+    s_progress_callback = progress_callback;
+}
+    
+enum class CSharpNotifierType : uint8_t {
```

Comment:
The OS type is `NotifierType`, so I wanted to make it clear we're proxying to that. I'm not completely certain if there's too much value to create our own enum, but figured I should follow the established pattern.